ghost/core/core/server/data/migrations/versions/6.0/2025-06-23-09-49-25-add-missing-member-uuids.js

```diff
@@ -0,0 +1,22 @@
+// For information on writing migrations, see https://www.notion.so/ghost/Database-migrations-eb5b78c435d741d2b34a582d57c24253
+
+const logging = require('@tryghost/logging');
+const uuid = require('uuid');
+
+// For DML - data changes
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        const membersWithoutUUID = await knex.select('id').from('members').whereNull('uuid');
+
+        logging.info(`Adding uuid field value to ${membersWithoutUUID.length} members.`);
+
+        // eslint-disable-next-line no-restricted-syntax
+        for (const member of membersWithoutUUID) {
```

Comment:
Do we have an idea of how many members this will be? It _should_ be zero as far as I understand - but it would be good to check this as we're doing a loop here and making a query for each item

Alternatively we could create the uuids up front and batch insert them

Reply:
On pro it's zero - this is only here for absolute edge cases in the wider world, and shouldn't cause any trouble